File: 2310817120009_AlysaArmelia_PemrogramanI_Modul5/Python/PRAK502-2310817120009-AlysaArmelia.py
def mutlak(angka):

    if angka < 0:

        angka *= -1

    return angka

def hitung(nilai1, nilai2):

    perhitungan = nilai1 - nilai2

    if perhitungan < 0:

        perhitungan *= -1
        
    return perhitungan

a, c, b, d = map(int, input().split())
Hasil = hitung(a, b) + hitung(c, d)
print(mutlak(Hasil))

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul5/Bahasa C/PRAK502-2310817120009-AlysaArmelia.c
#include <stdio.h>
#include <math.h>

int mutlak (int angka){
    
    if (angka < 0)
    {
        angka *= -1;
    }
    return angka;
}

int hitung (int nilai1, int nilai2){ 

    int Hasil;
    int perhitungan;

    perhitungan = nilai1 - nilai2;

    if (perhitungan < 0)
    {
        perhitungan *= -1;
    }
    return perhitungan;
} 

int main() 
{ 
    int a, b, c, d, Hasil;
    scanf("%d %d %d %d", &a, &c, &b, &d); 
    Hasil = hitung(a,b) + hitung(c,d); 
    printf("%d",mutlak(Hasil)); 
return 0; 
}